css/style.css

```diff
@@ -1146,6 +1142,20 @@ body::-webkit-scrollbar {
   transition: all 400ms ease-out;
 }
 
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+::-webkit-scrollbar-track {
+  background: rgba(0,0,0,0);
+  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0);
+}
+
+::-webkit-scrollbar-thumb {
+  background: rgba(107,107,107,0.6);
+  border-radius: 10px;
+}
+
 .outer .vertical-collapse,
```

Comment:
Using webkit scrollbars to test the usability of the solution. Should be replaced with cross-browser version